nautobot/core/templates/components/panel/footer_content_table.html

```diff
@@ -4,11 +4,34 @@
     </a>
 {% endif %}
 
+<div class="pull-left">
+    {% for button_context in buttons_context %}
+        <button form="object-table-form" type="button" class="btn btn-{{button_context.color}} primary btn-xs" action-url="{{button_context.link}}">
+            {{button_context.label}}
+        </button>
```

Comment:
This feels weird to me:
1) contexts are not specific to individual buttons?
2) why aren't we just using `button.render()`?